common/djangoapps/student/tests/test_models.py

```diff
@@ -403,3 +403,119 @@ def test_retire_recovery_email(self):
 
         # Assert that there is no longer an AccountRecovery record for this user
         assert len(AccountRecovery.objects.filter(user_id=user.id)) == 0
+
+
+@ddt.ddt
```

Comment:
Also open to testing more cases if anyone can think of any more that should get tested